MFVNeutralino/python/Vertexer_cfi.py

```diff
@@ -26,9 +51,11 @@
                                no_track_cuts = cms.bool(False),
                                min_seed_jet_pt = cms.double(30),
                                min_track_pt = cms.double(1),
+                               min_track_pt_loose = cms.double(0.9),
                                min_track_dxy = cms.double(0),
                                min_track_sigmadxy = cms.double(0),
                                min_track_rescaled_sigmadxy = cms.double(4),
+                               min_track_rescaled_sigmadxy_loose = cms.double(3.5),
```

Comment:
Are the loose ones actually used in the code currently? Maybe they're only used for something specific (like the track attachment), I haven't made it to closely looking at the Vertexer changes yet. If that is the case, I think we might want to either change the name to reflect that it's only used in a particular option, or put a comment.

Reply:
Those loose ones are just used when `match_jets` are set to be true. It's something similar to track attachment. I will add comments on it.